### PR TITLE
Scripts for working with db

### PR DESCRIPTION
**Pull request by alpdk**
There you can find some scripts for working with scripts, check them please.

### CODE REVIEW

File: traffic_info.sh
Comment (InnocentusLime) [inline traffic_info.sh:1]: Looks like `create_traffic.sh` would be a better name

**Review by InnocentusLime — CHANGES_REQUESTED** [review]
Looks good. Just rename one script :)

File: traffic_info.sh
Comment (alpdk) [reply traffic_info.sh:1]: I am not sure, because we are not creating traffic we are making logs for traffic that people use.

**Review by InnocentusLime — APPROVED** [review]
Excellent. Merging.

**Pull request merged**